src/components/simulation/SavingsStep.vue

```diff
@@ -2,18 +2,52 @@
 import InformationsEnergySaving from '@/components/simulation/InformationsEnergySaving.vue'
 import ElectricityConsumptionManual from '@/components/simulation/ElectricityConsumptionManual.vue'
 import { useSimulationStore } from '@/stores/simulations'
-import ElectricityConsumptionButton from './ElectricityConsumptionButton.vue'
+import ElectricityConsumptionButton from '@/components/simulation/ElectricityConsumptionButton.vue'
 import InformationsLinky from '@/components/simulation/InformationsLinky.vue'
+import AllowAnnualConsumptionPopup from '@/components/simulation/AllowAnnualConsumptionPopup.vue'
+import { ref } from 'vue'
+import { useConsumptionAndProductionStore } from '@/stores/consumptionAndProduction'
 
 const simulationStore = useSimulationStore()
+const consumptionAndProductionStore = useConsumptionAndProductionStore()
+const isDisplayPopup = ref(false)
+const stepAnnualConsumption = ref('manual')
+
+function showPopUpAllowAnnualConsumption(event: 'manual' | 'linky') {
+  stepAnnualConsumption.value = event
+  isDisplayPopup.value = true
+}
+
+function clickContinuePopup() {
+  isDisplayPopup.value = false
+  if (stepAnnualConsumption.value === 'manual') {
+    simulationStore.currentSubStep = 3
+  } else if (stepAnnualConsumption.value === 'linky') {
+    simulationStore.currentSubStep = 4
+  }
+}
+
+function goToEndSimulation() {
+  isDisplayPopup.value = false
+  consumptionAndProductionStore.setAnnualConsumption(6000)
```

Comment:
You can use this const from simulation store : 
![image](https://user-images.githubusercontent.com/126822753/233305673-b3a0f231-6c30-4ad3-919d-d0b1d2263858.png)